src/styles/blocks/product_item.scss

```
@@ -0,0 +1,91 @@
+.product-item {
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  width: $card-width;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__photo {
+    display: flex;
+    justify-content: center;
+    margin-bottom: 40px;
+  }
+
+  &__img {
+    height: 134px;
+    width: 160px;
+  }
```

It's enough to set only height or width for images
```suggestion
  &__img {
    height: 134px;
  }
```